examples/gno.land/p/gov/dao/proposal.gno

```diff
@@ -0,0 +1,47 @@
+package dao
+
+import "std"
+
+type Status string
+
+// ACTIVE -> ACCEPTED -> EXECUTED
+// ACTIVE -> NOT ACCEPTED
+var (
+	Active      Status = "active"       // proposal is still active
+	Accepted    Status = "accepted"     // proposal gathered quorum
+	NotAccepted Status = "not accepted" // proposal failed to gather quorum
+	Executed    Status = "executed"     // proposal is executed
+)
+
+func (s Status) String() string {
+	return string(s)
+}
+
+// PropStore defines the proposal storage abstraction
+type PropStore interface {
+	// GetProposals returns the given paginated proposals
+	GetProposals(offset, count uint64) []Proposal
+
+	// GetProposalByID returns the proposal associated with
+	// the given ID, if any
+	GetProposalByID(id uint64) (Proposal, error)
+
+	// GetProposalsByAddress returns the proposals associated
+	// with the given proposer address
+	GetProposalsByAddress(address std.Address) []Proposal
```

Comment:
what is this helper useful for?

Reply:
I agree, it's more bloat -- we can easily fetch this info through an indexer

Dropped it:
[8c45039](https://github.com/gnolang/gno/pull/2581/commits/8c45039a8c165b5dd4232acc410e601eea3908eb)